openapi/openapi.yml

```diff
@@ -2432,6 +2432,9 @@ components:
                   unix_user:
                     description: Unix user of the VM
                     type: string
+                  gpu:
```

Comment:
What if we return an array of objects like this: `gpu: [{"name": "NVIDIA A100 80GB PCIe"}, {"name": "NVIDIA A100 80GB PCIe"}]`?

Can the GPU device provide more information in the future?

Reply:
I thought about that, too. but right now you can create a vm with up to 8 identical gpus and we'd always return an array with identical items. I agree it's not optimal as is but might require a little more thought altogether.